linter_exclusions.yml

```diff
@@ -3960,3 +3960,13 @@ webapp update:
     skip_dns_registration:
       rule_exclusions:
       - option_length_too_long
+synapse spark pool create:
+  parameters:
+    enable_dynamic_executor_allocation:
+      rule_exclusions:
+      - option_length_too_long
+synapse spark pool update:
+  parameters:
+    enable_dynamic_executor_allocation:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
Can you add short options for these parameters instead of adding linter exclusion?